tests/test_runner/test_utils.py

```diff
@@ -21,7 +27,10 @@ def test_set_random_seed():
     b_np_random = np.random.rand(2, 2)
     b_torch_random = torch.rand(2, 2)
     assert torch.backends.cudnn.deterministic is True
-    assert torch.backends.cudnn.benchmark is False
+    if is_rocm_pytorch:
```

Comment:
the code should be moved to mmcv/runner/utils.py

https://github.com/open-mmlab/mmcv/blob/db580dda8d2bf4c0e3aee3639e053cef6c58b80f/mmcv/runner/utils.py#L58-L81

Reply:
@zhouzaida  because "Disabling benchmark mode for MIOpen is NOT supported. Overriding value to True"  so  `torch.backends.cudnn.benchmark = False`  could not take ettect ,see this 
[https://github.com/pytorch/pytorch/blob/4036820506693b71a96b9e20989bfe286acccd89/torch/csrc/Module.cpp#L509](url)